egs2/README.md

```diff
@@ -23,6 +23,8 @@ See: https://espnet.github.io/espnet/espnet2_tutorial.html#recipes-using-espnet2
 | dirha_wsj               | Distant-speech Interaction for Robust Home Applications                                 | Multichannel ASR        | EN                    | https://dirha.fbk.eu/, https://github.com/SHINE-FBK/DIRHA_English_wsj                                        |              |
 | dns_ins20               | Deep Noise Suppression Challenge – INTERSPEECH 2020                                     | SE                      | 7 languages + singing | https://www.microsoft.com/en-us/research/academic-program/deep-noise-suppression-challenge-interspeech-2020/ |              |
 | fsc                     | Fluent Speech Commands Dataset                                                          | SLU                     | EN                    | https://fluent.ai/fluent-speech-commands-a-dataset-for-spoken-language-understanding-research/               |              |
+| fsc_unseen              | Fluent Speech Commands Dataset MASE Eval splits                                         | SLU                     | EN                    | https://github.com/maseEval/mase                                                                             |              |
```

Comment:
What is MASE?

Reply:
MASE (Multi-Aspect Subtask Evaluation) is the name that we gave to our methodology for generating splits as shown here - https://maseeval.github.io/

Reply:
I will explain a bit more about this in the respective Readme for each split